src/Microsoft.DotNet.Tools.Compiler/CompilerUtil.cs

```diff
@@ -113,6 +115,46 @@ public static string GetResourceFileMetadataName(Project project, KeyValuePair<s
             return name;
         }
 
+        //used in incremental compilation
+        public static IEnumerable<string> GetCompilationSources(ProjectContext project) => project.ProjectFile.Files.SourceFiles;
+
+
```

Comment:
Probably a remnant from moving a lot of code around. Thanks for pointing that out